### PR TITLE
Update dockerfiles 17

### PR DESCRIPTION
**Pull request by pierallard**
The goal of this PR is to:
- drastically decrease the image file (from 1.56Go to 698Mo)
- create less useless layers
- avoid updating at several places the "1.X" string on version tag
- fix a bug on sensio vendors when deploying on prod

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,53 +1,84 @@
...
+    wget -O /tmp/sphinx-php.zip \
+        https://github.com/fabpot/sphinx-php/archive/v1.0.10.zip && \
+    unzip /tmp/sphinx-php.zip -d /tmp/ && \
+    cd /tmp/sphinx-php-1.0.10/ && \
+    python setup.py install && \
+    #
+    # Download packages
+    apt-get update && \
+    apt-get install -y --no-install-recommends mysql-server php7.1-apcu php7.1-bcmath \
+        php7.1-cli php7.1-curl php7.1-fpm php7.1-gd php7.1-intl php7.1-mcrypt php7.1-mysql php7.1-soap php7.1-xml \
```
Comment (damien-carcel) [inline Dockerfile]: PHP 7.1 for a PIM 1.x? Shouldn't you use 5.6?

File: Dockerfile [hunk 2]
```diff
@@ -1,53 +1,84 @@
...
     cd /home/akeneo/pim-docs/pim-community-dev-1.7/ && php app/console doctrine:schema:create --env=prod && \
-    cd /home/akeneo/pim-docs/pim-community-dev-1.7/ && php app/console pim:installer:assets --env=prod
+    cd /home/akeneo/pim-docs/pim-community-dev-1.7/ && php app/console pim:installer:assets --env=prod && \
+    service mysql stop && \
+    #
+    # Clean
+    rm -rf /root/.composer/cache && \
+    cd /home/akeneo/pim-docs/pim-community-dev-1.7/ && ls | grep -v vendor | xargs rm -rf && \
+    cd /home/akeneo/pim-docs/pim-community-dev-1.7/vendor/ && ls | grep -v sensio | xargs rm -rf && \
+    rm /var/lib/mysql/ibdata1 /var/lib/mysql/ib_logfile1 /var/lib/mysql/ib_logfile0
```
Comment (damien-carcel) [inline Dockerfile]: missing EOF

File: Dockerfile (hunk 1 above)
Comment (pierallard) [reply Dockerfile]: We can use 7.1 or 5.6 for 1.7. I put 7.1 to avoid multiple layers through upper versions

Comment (damien-carcel) [reply Dockerfile]: OK then.

**Pull request closed without merging**